src/collections/_documentation/learn/context.md

```diff
@@ -84,3 +84,7 @@ Additionally you can provide arbitrary key/value pairs beyond the reserved names
 In addition to the structured context that Sentry understands, you can send arbitrary key/value pairs of data which will be stored alongside the event. These are not indexed and are simply used to add additional information about what might be happening:
 
 {% include components/platform_content.html content_dir='set-extra' %}
+
+**Be aware of maximum payload size** - There are times, when you may want to send the whole application state as extra data.
+This can be quite a large object, which can easily weigh more than 200kB. This 200kB is currently the maximum payload size of a single event you can send to Sentry.
```

Comment:
Possible improvement, up to you though.

```suggestion
This is not recommended as application state can be very large and easily exceed the 200kB maximum that Sentry has on individual event payloads.
```

Reply:
Those inline updated recommendations are so slick! Thanks!